### ISSUE TITLE
[Accessibility] Missing external links when redirecting to external url

### ISSUE THREAD

**Issue by E-Jacques** [angular/angular-update-guide#241]
It's [good pratice](https://www.digitala11y.com/external-links-in-or-out/) to indicate to the user, via an icon (see below), that he will be redirected to another site. 
Those indicators aren't present yet on [update.angular.io](https://update.angular.io).

![Mediamodifier-Design-Template](https://github.com/angular/angular-update-guide/assets/55321938/713fb03a-ad22-41fa-8b43-1ac3902c6758)
